DFC.ServiceTaxonomy.Neo4j/Models/NeoDriver.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DFC.ServiceTaxonomy.Neo4j.Models.Interface;
+using Neo4j.Driver;
+
+namespace DFC.ServiceTaxonomy.Neo4j.Models
+{
+    public class NeoDriver : INeoDriver
+    {
+        public NeoDriver(string type, IDriver driver, string? uri)
+        {
+            this.Driver = driver;
+            this.Type = type;
+            this.Uri = uri;
+        }
+
+        public IDriver Driver { get; set; }
+        public string Type { get; set; }
+        public string? Uri { get; set; }
```

Comment:
A null Uri doesn't seem to have any significance, so can we change the type here (and in the ctor params & interface) to a non-nullable string?

Reply:
Endpoint configuration that is passed in is a nullable string type so followed convention

Reply:
I know, but if we can remove the nullable here, only `NeoDriverBuilder` has to handle null before it builds `NeoDriver`, whereas if it stays nullable here, all the consumers of `NeoDriver` will have to handle a null `Uri`.